src/datastore/src/userstore.js

```diff
@@ -33,6 +35,54 @@ class UserStore extends NetworkStore {
   }
 
   /**
+   * Find all users. A query can be optionally provided to return
+   * a subset of all users or omitted to return all users.
+   *
+   * @param {Query} [query] Query used to filter entities.
+   * @param {Object} [options] Options
+   * @param {Boolean} [options.discover] Discover users.
+   * @return {Observable} Observable.
+   */
+  find(query, options = {}) {
+    // If `options.discover`, use
+    // [User Discovery](http://devcenter.kinvey.com/guides/users#lookup)
+    // instead of querying the user namespace directly.
+    if (options.discover === true) {
```

Comment:
Since this a `super` call to the implementation for `find()` on the `NetworkStore` do we need separate unit tests for the `UserStore`? The only change would be that the `pathname` is overridden. We could need 1 unit test to check this.